addOns/client/src/main/javahelp/org/zaproxy/addon/client/resources/help/contents/spider.html

```diff
@@ -29,5 +32,24 @@ <H1>Client Side Integration - Client Spider</H1>
 <li>Automation Framework <a href="automation.html">spiderClient</a> job
 </ul>
 
+<h3>Client Spider tab</h3>
+
+The Client Spider tab allows you start and monitor the Client Spider.
+It provides 3 sub tabs:
+
+<h4>Added Nodes</h4>
+
+These are the nodes that are have been added to the Client Map.
+
+<h4>Tasks</h4>
+
+These are the tasks that spider uses to crawl the application. 
+The tasks are updated when they are added to the task list, when they start running and when they complete.
+This allows you to understand what the Client Spider is actually doing much more clearly.
+
+<h4>Messages</h4>
+
+These are the HTTP(S) messages sent from the browsers that the client uses.
```

Comment:
Just noticed Tasks puts the Action as "Get" while Messages lists the Method as "GET" should those be consistent? (I'm looking at a very limited example right now, tasks only has a single entry)

Reply:
That was actually deliberate 😁 The task is the "Get" the URL, which the Method is "GET". They are not exactly the same thing - one is an action and one is the name of a method..